src/components/MaterialUI/Surfaces/ExpansionPanel.test.js

```diff
@@ -0,0 +1,114 @@
+import React from "react";
+import { mount } from "enzyme";
+import ExpansionPanel from "./ExpansionPanel";
+import ExpansionPanelMui from "@material-ui/core/ExpansionPanel";
+import ExpansionPanelSummary from "@material-ui/core/ExpansionPanelSummary";
+import ExpansionPanelDetails from "@material-ui/core/ExpansionPanelDetails";
+import ExpansionPanelActions from "@material-ui/core/ExpansionPanelActions";
+import { ExpansionPanelProps, ExpansionPanelActionsProps } from "./expansionPanelProps";
+
+describe("Expansion Panel", () => {
+	it("Renders Expansion Panel", () => {
+		var header = <p>Header</p>;
+		var content = <p>Content</p>;
+		var actions = <p>Actions</p>;
+		var component = (
+			<ExpansionPanel header={header} content={content} actions={actions} />
+		);
+		var mountedComponent = mount(component);
+		var expected = (
+			<ExpansionPanelMui>
+				<ExpansionPanelSummary>{header}</ExpansionPanelSummary>
+				<ExpansionPanelDetails>{content}</ExpansionPanelDetails>
+				<ExpansionPanelActions>{actions}</ExpansionPanelActions>
+			</ExpansionPanelMui>
+		);
+
+		expect(mountedComponent.containsMatchingElement(expected), "to be truthy");
+	});
+
+	it("Fails if expansionPanelProps has wrong type", () => {
+		var component = <ExpansionPanel />;
+		var mountedComponent = () => {
+			component.props.expansionPanelProps = "Wrong type";
+
+			return mount(component);
+		};
+		expect(mountedComponent, "to throw");
```

Comment:
Is there a way to test what is thrown?

Reply:
Yes there is but, I'm throwing just an error. It's possible to throw TypeError for those cases. However I would say it doesn't make sense. The main point here that developer will be able to see in console an error message which will explain him that he passed parameter with wrong type in that wrapper (message is the main tip - not an error type).

Reply:
Replaced Error with TypeError and adjust tests with this.